### PR TITLE
jacocoRootReport

### PR DESCRIPTION
**Pull request by ones1kk**

### CODE REVIEW

**Comment by worker-hub** [comment]
Good

**Review by worker-hub — COMMENTED** [review]
Good

File: build.gradle [hunk 1]
```diff
@@ -147,4 +146,22 @@ subprojects {
...
+    executionData.from = files(subprojects.jacocoTestReport.executionData)
+
+    reports {
+        html.enabled true
+        xml.enabled true
+        html.destination file("$buildDir/jacoco/report/index.html")
+        xml.destination file("$buildDir/jacoco/report/index.xml")
+    }
+
 }
```
Comment (worker-hub) [inline build.gradle:167]: review test

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -37,4 +37,4 @@ jobs:
       uses: codecov/codecov-action@v1
       with:
         token: ${{ secrets.CODECOV_TOKEN }}
-        file: ./build/reports/jacoco/test/jacocoTestReport.xml
+        file: .build/jacoco/report/index.xml
```
Comment (worker-hub) [inline .github/workflows/build.yml:40]: review test

File: build.gradle (hunk 1 above)
Comment (worker-hub) [reply build.gradle:167]: good

Comment (ones1kk) [inline build.gradle:167]: good

**Comment by worker-hub** [comment]
good

**Comment by ones1kk** [comment]
> Good

test

**Review by worker-hub — APPROVED** [review]
test

**Review by ones1kk — COMMENTED** [review]
test

**Pull request merged**